src/deluge/dsp/reverb/mutable/reverb.hpp

```diff
@@ -143,6 +147,12 @@ class Mutable : public Base {
 
 	[[nodiscard]] float getHPF() const { return hp_cutoff_val_; }
 
+	void setLPF(float f) {
+		lp_cutoff_val_ = f;
+		lp_cutoff_ = calcFilterCutoff(f);
```

Comment:
This will make the maximum cutoff 520 Hz which is probably not intended - it's specific to turning a 0-1 knob turn into an appropriate HPF setting of 20-520 Hz with an exponential curve

Reply:
you could move the scaler variables to be function parameters and use the same function for both?